core/src/main/java/org/ehcache/core/spi/cache/AbstractValueHolder.java

```diff
@@ -181,6 +181,6 @@ public long getId() {
 
   @Override
   public String toString() {
-    return "" + value();
+    return "" + invoke();
```

Comment:
Would we always want `toString` to materialize the value?  Wouldn't we want some way of reflecting that the value is, as yet, not materialized?